docs/best-practices/testing.md

```diff
@@ -0,0 +1,60 @@
+# Table of Contents
+
+<!-- toc -->
+
+- [Testing best practices](#testing-best-practices)
+  * [Assertions](#assertions)
+    + [Examples](#examples)
+      - [Checking for equality](#checking-for-equality)
+      - [Checking array size](#checking-array-size)
+  * [Polling](#polling)
+
+<!-- tocstop -->
+
+# Testing best practices
+
+## Assertions
+In Kas Fleet Manager, we make extensive usage of the [gomega](https://onsi.github.io/gomega/) framework.
```

Comment:
wdyt of adding a note here to import gomega like below to make it easier to read?

```
import (
  . "github.com/onsi/gomega"
)
```